pyseq.py

```diff
@@ -63,6 +64,15 @@
 # regex for matching numerical characters
 digits_re = re.compile(r'\d+')
 
+# default settings for view pairs
+view_pairs = json.loads(os.environ.get('PYSEQ_VIEWPAIRS','[["left","right"],["blue","green","yellow"],["l","r"],["gouche","droit"],["rt","lt"],["destra","sinistra"]]' ))
```

Comment:
Would it be better to follow env var conventions and do an os.pathsep separated list of lists (or tuples)?

```
export PYSEQ_VIEWPAIRS="(left,right):(red,blue,blue)"
```
```
view_pairs = os.environ.get("PYSEQ_VIEWPAIRS", "(l,r)").split(os.pathsep)
view_pairs = [i.strip("()").split(",") for i in view_pairs]
```